packages/xod-cli/src/messages.js

```diff
@@ -1,9 +1,66 @@
-export const libraryFoundAndInstalling = libQuery =>
-  `Library "${libQuery}" was found. Resolving missing dependencies...`;
+export default {
+  UNKNOWN_ERROR: ({ message }) => ({
+    title: 'Error',
+    note: message,
+    solution:
+      'The error has no formatter, which is a bug. Report the issue to XOD developers.',
+  }),
 
-export const dependencyResolved = libName => `> "${libName}"`;
+  TRIED_TO_OPEN_NOT_XOD_FILE: ({ path }) => ({
+    title: 'Invalid file path',
+    note: `Tried to open not a xod file: ${path}`,
+    solution: 'Try to change path to project',
```

Comment:
I am looking at the code and the note/solution seem incorrect. Is it more precise to say:

```
note: `File ${path} does not exist`
solution: 'Provide a valid path to either patch, project directory, or xodball'
```